cloudsync/smartsync.py

```diff
@@ -257,6 +258,22 @@ def _ent_to_smartinfo(self, ent: Optional[SyncEntry], local_dir_info: Optional[D
                            )
         return retval
 
+    def _ignore_ent(self, ent: SyncEntry) -> bool:
+        '''Returns True if Smartsync should ignore the ent during listdir/info'''
+        if not ent:
+            return True
+
+        # Ent has been trashed locally or remotely
+        if ent[LOCAL].exists in (TRASHED, MISSING) \
+                    or ent[REMOTE].exists in (TRASHED, MISSING):
```

Comment:
I think remotely trashed files shouldn't be excluded, because if it is showing up here then the file exists locally still

Reply:
Not necessarily. There could be no local information and exists would be `UNKNOWN`  and in that case we wouldn't want to return the trashed ent. We could add an additional check to ensure that's the case though